docs/topics/jvm/java-to-kotlin-interop.md

```diff
@@ -586,3 +585,42 @@ fun emptyList(): List<Nothing> = listOf()
 // is translated to
 // List emptyList() { ... }
 ```
+
+### Inline value classes
+
+<primary-label ref="experimental-general"/>
+
+Kotlin compiles [inline value classes](inline-classes.md) to use **unboxed representations**, which are often inaccessible from Java.
+For example, Java isn't able to call a constructor for the `PositiveInt` class:
+
+```kotlin
+@JvmInline value class PositiveInt(val number: Int)
```

Comment:
Add a new line before value to match the same sample below.
Also, could you add a non compiling java sample to highlight the benefit from `@JvmExposeBoxed`?